README.md

```diff
@@ -60,20 +65,20 @@ Download and use the OCI image for Docker/Podman/Toolbx:
 ###### Running the container
 Docker:
 ```
-# docker run -v /PATH_TO_REPO/neo/src:/root/neo/src --rm -it --entrypoint /bin/bash registry.gitlab.steamos.cloud/steamrt/sniper/sdk
+# docker run -v /PATH_TO_REPO/:/root/neo/ --rm -it --entrypoint /bin/bash registry.gitlab.steamos.cloud/steamrt/sniper/sdk
 $ cd /root/neo/src/
 ```
 
 Podman: 
 ```
-$ podman run -v /PATH_TO_REPO/neo/src:/root/neo/src --rm -it --entrypoint /bin/bash registry.gitlab.steamos.cloud/steamrt/sniper/sdk
+$ podman run -v /PATH_TO_REPO/:/root/neo/ --rm -it --entrypoint /bin/bash registry.gitlab.steamos.cloud/steamrt/sniper/sdk
 $ cd /root/neo/src/
 ```
 
 Toolbx: 
```

Comment:
`Toolbox`?

Reply:
> `Toolbox`?

Not sure I understand the question. Do you mean whether that's a typo on line 78? [Probably not](https://containertoolbx.org/), although I'm not familiar with this tool.